aws-amplify/docs
# Add cononical link tag

#### Description of changes:
- Add list of canonical urls based on their "route"
- Add some logic in `_app.tsx` to set the link with `rel='canonical'` based on if the current route is part of the list

To test:
1. Go to https://canonical-urls.d1ywzrxfkb9wgg.amplifyapp.com/android/tools/cli/commands/
2. Inspect the page source and verify that the canonical link is for the `javascript` platform

#### Related GitHub issue #, if available:

### Instructions

**If this PR should not be merged upon approval for any reason, please submit as a DRAFT**

Which product(s) are affected by this PR (if applicable)?

Which platform(s) are affected by this PR (if applicable)?

**Please add the product(s)/platform(s) affected to the PR title**

#### Checks




      _ref: MDX: `[link](https://link.com)` 
            HTML: `<a href="https://link.com">link</a>`_

### When this PR is ready to merge, please check the box below

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._
